src/features/RequestForm/index.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react'
 import { Row, Tabs } from 'antd'
 
-import MaterialIts from 'features/RequestMaterials'
-import RequestEvents from 'features/RequestEvents'
+import RequestMaterials from 'features/RequestForm/RequestMaterials'
+import RequestVacation from 'features/RequestForm/RequestVacation'
```

Comment:
use common export file for  'RequestMaterials' and 'RequestVacation'

Reply:
in 'RequestForm' index is used for component and in this case i think is not better